reference/5.1/Microsoft.PowerShell.Core/About/about_Classes.md

```diff
@@ -568,30 +567,31 @@ Brand Model
 ----- -----
 
 Exception setting "Brand": "The argument is null or empty. Provide an
-argument that is not null or empty, and then try the command again."
+argument that isn't null or empty, and then try the command again."
 At C:\tmp\Untitled-5.ps1:11 char:1
 + $dev.Brand = ""
 + ~~~~~~~~~~~~~~~
     + CategoryInfo          : NotSpecified: (:) [], SetValueInvocationException
     + FullyQualifiedErrorId : ExceptionWhenSetting
 ```
 
+For a complete list of available attributes, see
+[about_Functions_Advanced_Parameters][03].
```

Comment:
Nit: Not _technically_ a complete list - just the list of built-in attributes that are applied to parameters.

Maybe instead:

```suggestion
For more information on available attributes, see
[about_Functions_Advanced_Parameters][03].
```